ninja-core/src/main/java/ninja/Context.java

```diff
@@ -523,6 +526,36 @@ private HTTP_STATUS(int code) {
     FileItemIterator getFileItemIterator();
 
     /**
+     * Gets the uploaded file stream for the given key from the multipart
+     * request. When multiple files uploaded for the same key, the first file is
+     * returned.
+     *
+     * @param name the key of the file
+     * @return the input stream to read file contents from; {@code null} if no
+     * file uploaded for the given key
+     */
+    InputStream getUploadedFileStream(String name);
```

Comment:
I am thinking about not returning an InputStream, but NinjaFileItemStream. We could return a proper NinjaFileItemStream (a but stripped down so that it does not contain all methods from FileItemStream). If a user wants to copy a file to a proper location it is easy to do so, as well as getting the content type and then it is also clea that the outputstream has to be closed, too...

What do you think about that?